v22.1/alter-range-relocate.md

```diff
@@ -0,0 +1,228 @@
+---
+title: ALTER RANGE ... RELOCATE
+summary: Use the ALTER RANGE ... RELOCATE statement to move a lease or replica between stores in an emergency situation.
+toc: true
+docs_area: reference.sql 
+---
+
+<span class="version-tag">New in v22.1:</span> The `ALTER RANGE ... RELOCATE` statement is a subcommand of [`ALTER RANGE`](alter-range.html). It is used to move a lease or [replica](architecture/overview.html#architecture-replica) between [stores](cockroach-start.html#store). This is helpful in an emergency situation to relocate data in the cluster.
+
+{{site.data.alerts.callout_danger}}
+Most users should not need to use this statement; it is for use in emergency situations. If you are in an emergency situation where you think using this statement may help, Cockroach Labs recommends contacting [support](support-resources.html).
+{{site.data.alerts.end}}
+
+{% include {{ page.version.version }}/misc/schema-change-stmt-note.md %}
+
+## Synopsis
+
+<div>
+{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/release-22.1/grammar_svg/alter_range_relocate.html %}
+</div>
+
+## Required privileges
+
+To alter a range and move a lease or replica between stores, the user must have one of the following:
+
+- Membership to the [`admin`](security-reference/authorization.html#admin-role) role for the cluster.
+
+## Examples
+
+### Find the cluster node IDs and store IDs
+
+To use `ALTER RANGE ... RELOCATE`, you will need to know your cluster's node IDs and store IDs. To get the node IDs and store IDs in your cluster, run the following statement:
```

Comment:
The commands work purely on store id's so you dont need node id's for anything.

Reply:
Fixed by removing mention of node IDs